library-backend/src/main/java/pl/edu/pw/ee/library/api/book/BookServiceImpl.java

```diff
@@ -61,13 +62,13 @@ public final List<BookResponse> searchByBookName(String bookName) {
 
         List<Book> bookList = bookRepository.findByTitle(bookName);
 
-        if(bookList.isEmpty()) {
-            throw new BookNotFoundException(String.format("There are no books matching title '%s'", bookName));
-        }
+        List<BookResponse> bookResponseList = bookList.stream()
+                        .map(bookMapper::toBookResponse)
+                        .toList();
 
         log.info("Returning bookResponseList of : {}", bookList);
```

Comment:
Tutaj bardziej loguj liczbę tych książek co zwracasz bo jak ci lognie 100 książek to trochę tego log będzie długi.